src/activity_handlers.cpp

```diff
@@ -2564,10 +2565,19 @@ void repair_item_finish( player_activity *act, Character *you, bool no_menu )
             }
         }
 
-        title += string_format( _( "Charges: <color_light_blue>%s/%s</color> %s (%s per use)\n" ),
-                                ammo_remaining, used_tool->ammo_capacity( current_ammo, true ),
-                                ammo_name,
-                                used_tool->ammo_required() );
+        std::string charges_text = string_format(
+                                       _( "Charges: <color_light_blue>%s/%s</color> %s (%s per use)\n" ),
+                                       ammo_remaining, used_tool->ammo_capacity( current_ammo, true ),
+                                       ammo_name,
+                                       used_tool->ammo_required() );
+
+        if( used_tool->is_tool() && used_tool->has_flag( flag_USES_NEARBY_AMMO ) ) {
+            // Remove "/capacity" from the "ammo/capacity" pattern, since the tool doesn't hold the ammo.
+            std::regex pattern( R"((\d+)/0)" );
+            charges_text = std::regex_replace( charges_text, pattern, "$1" );
```

Comment:
This is actually not a good idea because there's no guarantee that the translated string will have the same format as the original string.

Reply:
@Qrox Good point. I know of at least one language that loves putting spaces between their symbols, for one.
@Uwuewsky Alright, thanks. I added an additional string for the new flag.